youtube_transcript_api/test/test_cli.py

```diff
@@ -50,6 +50,15 @@ def test_argument_parsing(self):
         self.assertEqual(parsed_args.https_proxy, 'https://user:pass@domain:port')
         self.assertEqual(parsed_args.http_proxy, '')
 
+        parsed_args = YouTubeTranscriptCli(
+            'v1 v2 --languages de en --json'.split()
```

Comment:
These arguments are equivalent to the second testcase within `test_argument_parsing`. However I think the expected values for `parsed_args.http_proxy` and `parsed_args.https_proxy` should be added to all testcases.